### ISSUE TITLE
panlint: ssh keys - line too long

### ISSUE THREAD

**Issue by wdpypere** [quattor/pan#141]
As I started using panlint to cleanup our tree I found it fails for ssh authorized keys on line too long.
Now I could not find a way to exclude them, or split them in a functional way to keep panlint happy as well as maintain the expected output in the profile.

**Comment by jrha** [comment]
We encounter the same problem and simply ignore the errors for those files, do you have a better idea for how to handle this?

**Comment by wdpypere** [comment]
well, we run panlint as a test on changes so having a --exclude option or something and then a regex for something like "ssh-rsa <high number of characters>" ?

**Comment by wdpypere** [comment]
We now load the ssh keys in a different way, so this is no longer an issue for us. afaik, this can be closed.

**Comment by ned21** [comment]
We've had this problem with a couple of other lines where we are embedding content. How about a generic #nolint<N> pragma to tell panlint to ignore the <N> following lines?

**Comment by gombasg** [comment]
Yup, I've seen `panlint` complaining path literals should not have trailing slashes - except those strings were _not_ Pan path literals.

No static analyzers can ever be perfect, so having a way for suppressing certain warnings is necessary.

**Comment by jrha** [comment]
Could you take a look at #220? The rest of the changes that follow the re-factoring work there provide such functionality.

**Comment by ned21** [comment]
@jrha I am happy with #220 but it depends on #219 where I hadn't reviewed the regexes in detail to be confident of merging it. And now we have #222 which will probably invalidate both of them. In which order do we proceed?

**Comment by wpoely86** [comment]
It's not that much work to redo #222 on top of the other two if that helps.
